composables/transaction/transactionOffer.ts

```diff
@@ -0,0 +1,28 @@
+import { getExpiration } from '@/utils/api/bsx/query'
+import { calculateBalance } from '@/utils/format/balance'
+import { bsxParamResolver, getApiCall } from '@/utils/gallery/abstractCalls'
+import { dangerMessage } from '@/utils/notification'
+import { ShoppingActions } from '@/utils/shoppingActions'
+
+import type { ActionOffer } from './types'
+
+export async function execTsxOffer(item: ActionOffer, api, executeTransaction) {
```

Comment:
execTsxOffer

`exec{What}Tx` ?

```suggestion
export async function exectMakeOfferTx(item: ActionOffer, api, executeTransaction) {
```